Tests/scripts/server_get_artifact.sh

```diff
@@ -18,13 +18,13 @@ if [[ "$ARTIFACT_BUILD_NUM" = "" ]]; then
     exit 1
 fi
 
-SERVER_DOWNLOAD_LINK=$(curl -s -H "$ACCEPT_TYPE" ${SERVER_API_URI}/${ARTIFACT_BUILD_NUM}/artifacts?${TOKEN_ATTR} | jq '.[].url' -r | grep demistoserver | grep /0/)
+SERVER_DOWNLOAD_LINK=$(curl -s -H "$ACCEPT_TYPE" ${SERVER_API_URI}/${ARTIFACT_BUILD_NUM}/artifacts?${TOKEN_ATTR} | jq '.[].url' -r | grep demistoserver | grep /0/ | head -n 1)
```

Comment:
Why are those changes?

Reply:
those are changes I have made, without them the build is failing(Still waiting for the build to pass to add them)